pkg/kubelet/dockertools/docker.go

```diff
@@ -812,3 +813,65 @@ type ContainerCommandRunner interface {
 	ExecInContainer(containerID string, cmd []string, in io.Reader, out, err io.WriteCloser, tty bool) error
 	PortForward(podInfraContainerID string, port uint16, stream io.ReadWriteCloser) error
 }
+
+// Parse the pod full name. TODO(yifan): This is duplicated with kubelet.ParsePodFullName.
+func parsePodFullName(podFullName string) (string, string, error) {
```

Comment:
Yes I think so too :) the parsing and unparsing of the podFullName should only happen in one place.